src/collections/_documentation/platforms/android/gradle-advanced.md

```diff
@@ -0,0 +1,67 @@
+# Advanced gradle plugin configuration
+
+If for some reason the gradle plugin isn't an option for you, here are other ways to get the tasks done:
+
+### Manual Integration
+
+If you choose not to use the Gradle integration, you may handle the processing and upload steps manually. However, we highly recommend that you use the Gradle integration, if at all possible.
+
+### ProGuard UUIDs
+
+After ProGuard files generate, you will need to embed the UUIDs of the ProGuard mapping files in a properties file named `sentry-debug-meta.properties` in the assets folder. The Java SDK will look for the UUIDs there to link events to the correct mapping files on the server-side.
+
+Sentry calculates UUIDs for ProGuard files. For more information about how this works, see [ProGuard UUIDs]({%- link _documentation/workflow/debug-files.md -%}#proguard-uuids).
+
+`sentry-cli` can write the `sentry-debug-meta.properties` file for you:
+
+```bash
+sentry-cli upload-proguard \
+    --android-manifest app/build/intermediates/manifests/full/release/AndroidManifest.xml \
+    --write-properties app/build/intermediates/assets/release/sentry-debug-meta.properties \
+    --no-upload \
+    app/build/outputs/mapping/{buildVariant}/mapping.txt
```

Comment:
those folders are looked up dynamically, not sure if we should add as a static path, it might create more questions.
eg `manifests` is called `merged_manifests` on newer versions
`assets` is called `merged_assets` on newer versions
where is `release`, it should be `{buildVariant}` because it's also dynamic.
I'd say that we should stick to a generic path and point them to the Gradle integration source code?

Reply:
Should I just delete this whole `advanced-gradle` doc?
I believe we should point users to the plugin only. Is it a real use-case that ppl go straight to calling a CLI from gradle instead of the plugin?

Reply:
I'd be in to kill it all, I really don't see a reason to have this manual steps.
a valid point would be using another build system instead of Gradle, but then again, those scripts won't work.
See: https://stackoverflow.com/questions/60282834/gradle-task-for-sentry-not-compiling
I just asked why people were trying to use those scripts instead of the Gradle integration and found no valid reason rather the Integration didn't work.
I think we should focus on making the integration to work smoothly rather than fixing those scripts.